lagom13-scala-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/scaladsl/ConductRApplicationComponents.scala

```diff
@@ -40,7 +40,7 @@ trait ConductRServiceLocatorComponents extends BundlelibComponents with LagomSer
   def actorSystem: ActorSystem
   def configuration: Configuration
 
-  lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
+  override lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
```

Comment:
This override is necessary because the lagom version was bumped to `1.3.5`. With this override `conductr-lib` after this commit won't be usable by `pre-1.3.3` lagom users.

I think `conductr-lib` should depend on  Lagom `1.3.0` and let users bring a later version. With that approach the code in `ConductRServiceLocatorComponents` should not use an `override` and it'd work with any version of Lagom `1.3.x` the user brought in.